abs/index.js

```diff
@@ -20,6 +80,43 @@ class Lib {
   afterEach () {
     throw new Error('Function afterEach is missing!')
   }
+
+  loadStubs (keys) {
+    keys.forEach((key) => {
+      stubs[key].forEach((stub) => {
+        sinon.stub(this.ledger, stub.name).callsFake(stub.func.bind(null, this))
+      })
+    })
+  }
+
+  setState (newState) {
+    let oldState = this.state
+    this.state = newState
+
+    if (!newState.hasIn(['ledger', 'info'])) {
+      return
+    }
+
+    if (!oldState.hasIn(['ledger', 'info'])) {
+      oldState = oldState.setIn(['ledger', 'info'], Immutable.Map())
+    }
+
+    const newInfo = newState.getIn(['ledger', 'info'])
+
+    newInfo.keySeq().forEach((key) => {
+      const temp = newInfo.getIn([key])
+
+      if (Immutable.Map.isMap(temp) &&
```

Comment:
`if (Immutable.Map.isMap(temp) && temp.size === 0) {` can we do it like this?

Reply:
Definitely :) This was an oversight for sure